Dockerfile

```diff
@@ -1,18 +1,28 @@
-# https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a
-FROM registry.access.redhat.com/ubi8/python-38
+FROM registry.access.redhat.com/ubi8/python-39
 
-ENV PYTHONPATH=/opt/app-root/src/src:/opt/app-root/src/packages
+ENV PATH "$PATH:/opt/app-root/packages/bin/"
+ENV PYTHONPATH=/opt/app-root/src/src:/opt/app-root/packages
+
+COPY ./requirements.txt ./requirements.txt
+RUN pip3 install --upgrade -r ./requirements.txt -t ../packages
 COPY . .
-RUN pip3 install --upgrade -r ./requirements.txt -t ./packages
 
 USER 0
 
-ENV DOCKERIZE_VERSION v0.6.1
-RUN wget -q https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+RUN chown -R 1001:0 /opt/app-root
 
 # "default" user in the ubi container
 USER 1001
 
+WORKDIR /opt/app-root/src/
+
+RUN set -a; source ./.env.defaults; set +a; \
```

Comment:
This was just a simple sanity check in makefile to make sure the container was built ok. I don't think it should be there...

Reply:
And FYI, `.env` may look like it is bash and it is safe to source it, but it's not. In `.env` files are no special characters, and quotes are not supported, they are part of value. `FOO=;:(){ :|:& };:` is ok in `.env`, but you don't want to source it into bash, I guess :smile:

Reply:
I was sourcing the `.defaults` file, with only keys and no values, so our bash is still working :)

The idea was to avoid exception raise in `_config.py` file.

However, the right way (imho) of doing that is to change in `_config.py` the `os.environ` to `os.getenv` with a default value (`''`), so we would not get an exception when building the image (OTOH we'll not raise this exception when running the application, but I don't think this is a problem).

Do you think this is okay? (I already force-pushed the changes in `_config.py`).

Reply:
`getenv` is already used for all other env variables anyway, so I think it's ok.